dev/com.ibm.ws.jsonb_fat/fat/src/com/ibm/ws/jsonb/fat/JSONBContainerTest.java

```diff
@@ -36,6 +38,7 @@
 import web.jsonbtest.JohnzonTestServlet;
 
 @RunWith(FATRunner.class)
+@SkipForRepeat(EE9_FEATURES) // TODO: Enable this once cdi-3.0 is available
```

Comment:
Do we have some sort of issue to track enabling all of the skipped tests once cdi-3.0 is available? If we just have this TODO comment in the code I worry that it will be forgotten.

Reply:
@aguibert We have an issue to track all remaining work, https://github.com/OpenLiberty/open-liberty/issues/11633.  We'll need to modify it slightly once this PR merges in its final state.

Reply:
done, thanks!